portfolio/src/main/webapp/visualizer.js

```diff
@@ -4,23 +4,41 @@ google.charts.load('current', {
 });
 google.charts.setOnLoadCallback(drawRegionsMap);
 
-function drawRegionsMap() {
-  const data = google.visualization.arrayToDataTable([
-    ['City',   'Population', 'Area'],
-    ['Chicago', 2000000, 1285.31],
-    ['New York City', 5000000, 1400.31],
-    ['Los Angeles', 4000000, 1300],
-    ['Philadelphia', 1500000, 800],
-    ['Stamford', 30000, 900],
-  ]);
 
+function drawRegionsMap(i) {
```

Comment:
To avoid the check below, provide a default value in function signature.

```suggestion
function drawRegionsMap(i = 0) {
```